src/plugins/zeromqsend/zeromqsend.c

```diff
@@ -83,11 +83,13 @@ int elektraZeroMqSendGet (Plugin * handle ELEKTRA_UNUSED, KeySet * returned, Key
 	if (!strcmp (keyName (parentKey), "system:/elektra/modules/zeromqsend"))
 	{
 		KeySet * contract = ksNew (
-			30, keyNew ("system:/elektra/modules/zeromqsend", KEY_VALUE, "zeromqsend plugin waits for your orders", KEY_END),
+			32, keyNew ("system:/elektra/modules/zeromqsend", KEY_VALUE, "zeromqsend plugin waits for your orders", KEY_END),
```

Comment:
It's nice that you updated the count, but you actually don't have to. The way `ksNew (N, ...)` works is that:

- If `N == 0`: The other arguments `...` are completely ignored and an empty keyset is created.
- For `N != 0`: All of the following are done
  1. It allocates space for at least `N` keys
  2. It allocates space for at least 16 keys
  3. It allocates space for all the keys in `...` and adds them all
  
So the actual size of the keyset will be `N == 0 ? 0 : max (16, N, [no of keys in ...])`. Basically, unless you want to reserve space for extra keys added later, you can just use `ksNew (1, ...)`.